docs/Design/fpp-json-dict.md

```diff
@@ -131,11 +131,12 @@ Example: Type Descriptor for booleans
 ```
 
 ## String Type Descriptors
+
 | Field | Description | Options | Required |
 | ----- | ----------- | ------- | -------- |
 | `name` | **String** representing the FPP type name |  `string` | true |
 | `kind` | **String** representing the kind of type | `string` | true | 
-| `size` | **Number** of bytes supported by the data type | **Number** in the range [0, 2<sup>31</sup>) | false |
+| `size` | Maximum string size in bytes | **Number** in the range [0, 2<sup>31</sup>) | true |
```

Comment:
Looks good. Maybe the size description should say `**Number** representing the maximum string size in bytes`, for consistency. Also, zero is an invalid string size. The min should be 1.

Reply:
Actually the FPP spec does say zero is allowed, but this is wrong: file:///Users/bocchino/JPL/Tools/fpp/docs/spec/fpp-spec.html#Types_String-Types. The fpp-check tool excludes zero, and this is right.
```
[I] bocchino@MT-306179 ~> fpp-check
struct S { s: string size 0 }
fpp-check
stdin:1.27
struct S { s: string size 0 }
                          ^
error: invalid string size 0
```

Reply:
Fixed description and range starting at 1

Reply:
I'll fix the error in the spec in my next PR to FPP.